src/excel_templates/build.clj

```diff
@@ -174,21 +185,27 @@ If there are any nil values in the source collection, the corresponding cells ar
    This keeps all the styles and annotations while letting us write the
    spreadsheet using streaming so it can be arbitrarily large"
   [template-file output-file]
-  (let [tmpfile (File/createTempFile "excel-template" ".xlsx")]
+  (let [tmpfile (create-temp-xlsx-file "excel-template")]
     (try
       (io/copy template-file tmpfile)
-      (let [wb (XSSFWorkbook. (.getPath tmpfile))]
-        (doseq [sheet-num (range (.getNumberOfSheets wb))]
-          (let [sheet (.getSheetAt wb sheet-num)
-                nrows (inc (.getLastRowNum sheet))]
-            (doseq [row-num (reverse (range nrows))]
-              (when-let [row (.getRow sheet row-num)]
-                (.removeRow sheet row)))))
-        ;; Write the resulting output Workbook
-        (with-open [fos (FileOutputStream. output-file)]
-          (.write wb fos)))
+      (let [pkg (OPCPackage/open tmpfile)
+            wb (XSSFWorkbook. pkg)]
+          (doseq [sheet-num (range (.getNumberOfSheets wb))]
+            (let [sheet (.getSheetAt wb sheet-num)
+                  nrows (inc (.getLastRowNum sheet))]
+              (doseq [row-num (reverse (range nrows))]
+                (when-let [row (.getRow sheet row-num)]
+                  (.removeRow sheet row)))))
+          ;; Write the resulting output Workbook
+          (with-open [fos (FileOutputStream. output-file)]
+            (.write wb fos)
+
+            )
+          (.close pkg)
```

Comment:
Can you put this in the finally (doesn't really matter much, but it's better form